metaflow/tracing/__init__.py

```diff
@@ -0,0 +1,58 @@
+import sys
+from metaflow.metaflow_config import (
+    OTEL_ENDPOINT,
+    ZIPKIN_ENDPOINT,
+    CONSOLE_TRACE_ENABLED,
+)
+from functools import wraps
+import contextlib
+from typing import Dict
+
+
+def init_tracing():
+    pass
+
+
+@contextlib.contextmanager
+def post_fork():
+    yield
+
+
+def cli_entrypoint(name: str):
+    def cli_entrypoint_wrap(func):
+        @wraps(func)
+        def wrapper_func(*args, **kwargs):
+            return func(*args, **kwargs)
+
+        return wrapper_func
+
+    return cli_entrypoint_wrap
+
+
+def inject_tracing_vars(env_dict: Dict[str, str]) -> Dict[str, str]:
+    return env_dict
+
+
+def get_trace_id() -> str:
+    return ""
+
+
+@contextlib.contextmanager
+def traced(name, attrs={}):
+    yield
+
+
+def tracing(func):
+    @wraps(func)
+    def wrapper_func(*args, **kwargs):
+        return func(*args, **kwargs)
+
+    return wrapper_func
+
+
+if CONSOLE_TRACE_ENABLED or OTEL_ENDPOINT or ZIPKIN_ENDPOINT:
+    try:
+        from .tracing_modules import *
```

Comment:
Can we change this to an explicit import of those 7 functions? Arguably it would make it more clear for the reader that above dummy implementations get overridden.

Reply:
Pondering on this as well, as it caught me off guard while reviewing even though I was pushing for this approach a while ago :D There are definite benefits to this approach though.

Reply:
changed these to explicit imports for clarity.